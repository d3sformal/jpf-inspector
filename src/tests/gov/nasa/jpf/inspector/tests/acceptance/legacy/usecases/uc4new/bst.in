commence
st
st
print Transitions.#static
bst 2
print Transitions.#static
st
print Transitions.#static
st
print Transitions.#static
st
print Transitions.#static
bst 3
print Transitions.#static