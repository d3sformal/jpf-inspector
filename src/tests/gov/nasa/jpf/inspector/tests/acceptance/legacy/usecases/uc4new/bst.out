INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 hit_count<=1 instruction_type=any]
	SuT (Thread=0) executes the gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc4new/Transitions.java:0 - invokestatic gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc4new.Transitions.<clinit>()V
INFO: SuT is stopped
	SuT (thread 0) will now execute java/lang/Thread.java:-1 (executenative JPF_java_lang_Thread.start____V), source unavailable
INFO: SuT is stopped
	SuT (thread 1) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc4new/Transitions.java:0 (runstart), source unavailable
#thread[1].#stackFrame[0].Transitions.Transitions (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc4new.Transitions) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
  No instance fields
  Static fields
	0 : lastOperation (java.lang.String) = "MAIN: BEFORE JOIN"
INFO: SuT is stopped
	SuT (thread 0) will now execute java/lang/Thread.java:-1 (executenative JPF_java_lang_Thread.start____V), source unavailable
#thread[0].#stackFrame[0].Transitions.Transitions (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc4new.Transitions) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
  No instance fields
  Static fields
	0 : lastOperation (java.lang.String) = "MAIN: BEFORE START"
INFO: SuT is stopped
	SuT (thread 1) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc4new/Transitions.java:0 (runstart), source unavailable
#thread[1].#stackFrame[0].Transitions.Transitions (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc4new.Transitions) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
  No instance fields
  Static fields
	0 : lastOperation (java.lang.String) = "MAIN: BEFORE JOIN"
INFO: SuT is stopped
	SuT (thread 0) will now execute java/lang/Thread.java:-1 (executenative JPF_java_lang_Thread.join____V), source unavailable
#thread[0].#stackFrame[0].Transitions.Transitions (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc4new.Transitions) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
  No instance fields
  Static fields
	0 : lastOperation (java.lang.String) = "AUXILIARY: RUNNING"
INFO: SuT is stopped
	SuT (thread 1) will now execute gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc4new/Transitions.java:0 (runstart), source unavailable
#thread[1].#stackFrame[0].Transitions.Transitions (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc4new.Transitions) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
  No instance fields
  Static fields
	0 : lastOperation (java.lang.String) = "MAIN: BEFORE START"
ERR: There aren't that many transitions beginning with the specified choice generator type in the transition path.
#thread[1].#stackFrame[0].Transitions.Transitions (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc4new.Transitions) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
  No instance fields
  Static fields
	0 : lastOperation (java.lang.String) = "MAIN: BEFORE START"
INFO: JPF Terminating